pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -1500,6 +1501,8 @@ async def aspirate(
 
       hamilton_liquid_classes: Override the default liquid classes. See
         pylabrobot/liquid_handling/liquid_classes/hamilton/star.py
+      liquid_surface_no_lld: Liquid surface at function without LLD [0.1mm]. Must be between 0
+          and 3600. Default 3600.
```

Comment:
commenting for visibility to you: this should be 1mm, between 0 and 360

Reply:
i thought you were requesting changes but seems like you did it 😅

Reply:
I actually added a bit more

Reply:
of course, just making sure you use the correct units :)